input/write_reactors.py

```diff
@@ -184,7 +283,7 @@ def region_render(array, template, full_template, output_file):
         os.system('rm ' + country + '_region')
 
 
-def main(csv_file, reactor_template, region_template,
+def main(csv_file, reactor_template, deployinst_template,
```

Comment:
Yeah, I was wondering about that, If I should have the region only version (non -deployinst) of this script in a separate folder. Can I make another folder that is 'NullInst' only?